### ISSUE TITLE
macOS Ventura with MX Master 3 can't catch button input [bug]

### ISSUE THREAD

**Issue by NoirJ0e** [noah-nuebling/Mac-Mouse-Fix#410]
## Description

Mac Mouse Fix version: 3.0.0 Beta 3(18468)
macOS version: 13.0 (22A5373b)

Problem appear:

Update to beta 3 and lost all modify keys in settings, try to re-config it but the app can't catch button input except primary mouse button and secondary mouse button

## Steps to reproduce

1. Upgrade to macOS Ventura
2. Upgrade or reinstall mac mouse fix 3.0.0 beta 3
3. Navigate to Buttons page in settings
4. Click mouse button in   sign area
5. Nothing happened

## Context

Mac Mouse Fix Version: 3.0.0 Beta 3
macOS Version: Ventura 13.0 (22A5373b)
Mouse Model: Logitech MX Master 3

## Additional Comments

–

---

### Console Logs

https://pastebin.com/MSh0Sc3W

### Crash Reports

No crash happend

**Comment by noah-nuebling** [comment]
Hey @NoirJ0e, thanks for your feedback!

I've experienced similar issues. For me this was due to some bugs in the new Ventura APIs that Mac Mouse Fix calls when you try to enable it. These APIs break if you have multiple copies of the app in different locations. 

I could fix these issues with the following steps:

- Delete all versions of MMF from your computer
- Empty the Trash
- Restart your computer
- Reinstall Mac Mouse Fix

Let me know how it goes.If you still have issues after this I'll look into it further.

**Comment by NoirJ0e** [comment]
> Hey @NoirJ0e, thanks for your feedback!
> 
> I've experienced similar issues. For me this was due to some bugs in the new Ventura APIs that Mac Mouse Fix calls when you try to enable it. These APIs break if you have multiple copies of the app in different locations.
> 
> I could fix these issues with the following steps:
> 
> * Delete all versions of MMF from your computer
> * Empty the Trash
> * Restart your computer
> * Reinstall Mac Mouse Fix
> 
> Let me know how it goes.If you still have issues after this I'll look into it further.

After restart the computer it begin to work, perhaps somehow the upgrades inside the app doesn't kill all instance running in the background and met such problem, thx for your help

**Comment by noah-nuebling** [comment]
Glad to help!

I've ran into this a lot and I don't think it's something I can fix on my end unfortunately. I filed a bug report with Apple but they unfortunately haven't reacted so far and it's still an issue in the latest Ventura Beta (and it'll have it's official release next week)

If anyone wants to file more reports with Apple about this that should make it more likely that they focus on this.

Just put into your report that the `SMAppService ` API breaks when you have different versions of the same app on your computer, and maybe tell them more about your specific situation and ideally how they can reproduce the issue.

They encourage duplicate reports as far as I remember.
